config/non-serializable/mailer.js

```diff
@@ -3,4 +3,8 @@ module.exports = {
     host: `${process.env.MAILER_HOST || 'localhost'}`,
     port: `${process.env.MAILER_PORT || 25}`,
   },
+  auth: {
+    user: `${process.env.MAILER_AUTH_USER || ''}`,
```

Comment:
also, could this just be `MAILER_USER` and `MAILER_PASS`? Simplifies the pillar configuration to just be a `mailer` dictionary